circuit-lib/circuit-lib.js/tests/indices.test.ts

```diff
@@ -2,9 +2,23 @@ import { assert } from "chai";
 import { MINT, hashAndTruncateToCircuit } from "../../../zk.js/src/index";
 import { getIndices3D } from "../src/index";
 import { PublicKey } from "@solana/web3.js";
+import { it } from "mocha";
+const createBlakeHash = require("blake-hash");
+const { blake2b } = require("@noble/hashes/blake2b");
 
 //TODO: separate 3 dim indices template from light circuits in circuit-lib.circom and add similar test
 describe("Utxo Functional", () => {
+  
+  it("noble/blake vs blake-hash", async () => {
+    const b2params = { dkLen: 32 };
+    const input = "000";
+    const a = new Uint8Array(createBlakeHash("blake512").update(input).digest().slice(0, 32));
+    const b = blake2b.create(b2params).update(input).digest();
+
+    assert.equal(a.length, b.length);
+    assert.equal(a, b);
+  });
```

Comment:
Please move the test above into it's own file, so that the naming and contents of the test files is consistent.